pkg/proxy/winkernel/proxier.go

```diff
@@ -188,20 +187,24 @@ func (t DualStackCompatTester) DualStackCompatible(networkName string) bool {
 		return false
 	}
 
-	globals, err := hcn.GetGlobals()
-	if err != nil {
-		klog.ErrorS(err, "Unable to determine networking stack version. Falling back to single-stack")
-		return false
-	}
-
-	if !kernelSupportsDualstack(globals.Version) {
-		klog.InfoS("This version of Windows does not support dual-stack. Falling back to single-stack")
+	// First tag of hcsshim that has a proper check for dual stack support is v0.8.17, although it would've been hard to use that tag due
```

Comment:
nit but I think just saying 8.22 is the version of hcssim required due to a bug is enough in the comment.

Reply:
Done